### PR TITLE
Added method of Capture instance for v2

### PR DESCRIPTION
**Pull request by bhagyagkwd**
Jira:-  https://jsw.ibm.com/browse/POWERIAAS-4837

### CODE REVIEW

File: examples/capture-instance/main.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	//session Inputs
+	token := " < IAM TOKEN > "
+	region := " < REGION > "
+	accountID := " < ACCOUNT ID > "
+	zone := " < ZONE > "
+
+	// volume inputs
+	piID := "< POWER INSTANCE ID >"
+	volumes := make([]string, 1)
+	volumes[0] = "ef82d430-4cb0-46a5-be11-e61fb129fe18"
```
Comment (yussufsh) [inline examples/capture-instance/main.go:23]: Refrain from providing the data values.

Comment (bhagyagkwd) [reply examples/capture-instance/main.go]: done

File: examples/capture-instance/main.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	CloudStorageRegion := "< Cloud Storage Region >"
+
+	session, err := ps.New(token, region, true, accountID, zone)
+	if err != nil {
+		log.Fatal(err)
+	}
+	powerClient := v.NewIBMPIInstanceClient(context.Background(), session, piID)
+	if err != nil {
+		log.Fatal(err)
+	}
```
Comment (yussufsh) [inline examples/capture-instance/main.go:41]: Why checking err again?

File: examples/capture-instance/main.go [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+	body := &models.PVMInstanceCapture{
+		CaptureDestination:    &captureDestination,
+		CaptureName:           &captureName,
+		CaptureVolumeIds:      volumes,
+		CloudStorageAccessKey: CloudStorageAccessKey,
+		CloudStorageImagePath: CloudStorageImagePath,
+		CloudStorageRegion:    CloudStorageRegion,
+		CloudStorageSecretKey: CloudStorageSecretKey,
+	}
+	createRespOk, err := powerClient.CaptureInstanceToImageCatalogV2(instance_name, body)
```
Comment (yussufsh) [inline examples/capture-instance/main.go:52]: It will be good if we are able to delete the image from catalog to clean up the data at the end.

**Review by yussufsh — APPROVED** [review]

**Review by yussufsh — APPROVED** [review]

**Pull request merged**